README.md

```diff
@@ -1,12 +1,9 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+# This repo contains [Joachim Isaac's](https://ssh.cloud.google.com/cloudshell/editor) portfolio and SPS projects.
 
 To get started:
 
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+Login to GitHub
+Clone this repo: by clicking the clone button
+If your using git through your terminal use: git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY
```

Comment:
Update the URL to point to this repo: https://github.com/JoachimIsaac/software-product-sprint